assets/templates/partials/table-of-contents.tmpl

```diff
@@ -3,17 +3,17 @@
 <aside class="ons-toc-container" role="complementary">
   <nav
     class="ons-toc"
-    {{ if .TableOfContents.AriaLabelLocaliseKey }}
-      aria-label="{{ localise .TableOfContents.AriaLabelLocaliseKey .Language 1 }}"
+    {{ if .TableOfContents.AriaLabel.LocaleKey }}
+      aria-label="{{ localise .TableOfContents.AriaLabel.LocaleKey .Language .TableOfContents.AriaLabel.Plural }}"
     {{ else }}
-      aria-label="{{ .TableOfContents.AriaLabel }}"
+      aria-label="{{ .TableOfContents.AriaLabel.Text }}"
     {{ end }}
   >
     <h2 class="ons-toc__title ons-u-fs-r--b ons-u-mb-s">
-      {{ if .TableOfContents.TitleLocaliseKey }}
-        {{ localise .TableOfContents.TitleLocaliseKey .Language 1 }}
+      {{ if .TableOfContents.Title.LocaleKey }}
+        {{ localise .TableOfContents.Title.LocaleKey .Language .TableOfContents.Title.Plural }}
       {{ else }}
-        {{ .TableOfContents.Title }}
+        {{ .TableOfContents.Title.Text }}
       {{ end }}
     </h2>
     <ol class="ons-list ons-u-mb-m ons-list--dashed">
```

Comment:
You've updated the model but not updated the template file (github won't let me comment on the correct line) 

`line 29` should be updated to something like
```tmpl
 <a href="#{{ $id }}" class="ons-list__link">
  {{ if $section.Title.LocaleKey }}
    {{ localise $section.Title.LocaleKey $.Language $section.Title.Plural }}
  {{ else }}
    {{ $section.Title }}
  {{ end }}
</a>
```